recognition:
  # Model Preprocessing
  descr_rad: 0.06
  k_nearest_neighbors: 10
  down_sample: 0.002
  key_point_sampling: 0.01
  #ICP
  min_sample_distance: 0.02
  max_correspondence_distance: 0.01
  nr_iterations: 500
  #correspondence grouping
  cg_size: 0.01
  cg_thresh: 4.0
  descr_dis_thrd: 0.1
  # fine_align
  iteration: 10
  # other options
  switches:
    ICP: false # if false, use correspondence grouping algorithm
    print_detailed_info: true
  
# Segmentation  
segmentation:
  # general
  down_sample: 0.004
  # clustering
  min_cluster_size: 100
  max_cluster_size: 50000
  cluster_tolerance: 0.1
  # passthrough filtering
  camera_roi: # relative to the camera frame
    x: [-0.22543, 0.192618]
    y: [-0.2874, 0.283763]
    z: [0.0, 0.7162]
  # other options
  switches:
    print_detailed_info: true

